cdap-data-fabric/pom.xml

```diff
@@ -156,6 +156,8 @@
     <dependency>
       <groupId>org.iq80.leveldb</groupId>
       <artifactId>leveldb</artifactId>
+      <!-- leveldb 0.12 has conflicting guava 20 as a dependency. The uber jar shades guava -->
+      <classifier>uber</classifier>
```

Comment:
I think you only need to specify it once in the top level pom?

Reply:
was getting errors about no version being found without this

Reply:
Got it